controllers/keda/kedacontroller_controller.go

```diff
@@ -401,6 +411,52 @@ func (r *KedaControllerReconciler) installMetricsServer(ctx context.Context, log
 		logger.Info("Not running on OpenShift -> using generated self-signed cert for KEDA Metrics Server")
 	}
 
+	// Audit logging validation - configMap exists, logOutVolumeClaim validation
+	// policy is a wrapper AuditPolicy for auditv1.Policy for easier user exposure
+	policy := instance.Spec.MetricsServer.AuditConfig.Policy
```

Comment:
what happens if `AuditConfig` is nil? does the code works?

Reply:
if nothing in  `AuditConfig` is specified it just behaves as empty - the struct itself has fields that are nil (or default value) - this works when no audit logging is desired by user.